app/controllers/admin/users_controller.rb

```diff
@@ -2,12 +2,12 @@ class Admin::UsersController < Admin::BaseController
 
   def index
     authorize(User)
-    @users = policy_scope(User.member).page(params[:page])
+    @users = sort(policy_scope(User.member).page(params[:page]))
```

Comment:
it's just a scope so it doesn't matter in which order you do it. I'll change it around so it reads better, though